packages/react/src/theme/default-theme.ts

```diff
@@ -99,6 +99,7 @@ export const generateDefaultThemeOptions = (baseTheme: Theme): RecursivePartial<
     MuiCard: {
       styleOverrides: {
         root: {
+          borderColor: '#d7d9e3',
```

Comment:
Can't we use `--oxygen-palette-divider` here?

Reply:
The card border color in the design is different from the divider color.